cmd/build/helmify/static/values.yaml

```
@@ -290,6 +294,13 @@ audit:
       - 999
   writeToRAMDisk: false
   extraRules: []
+  disableWebhookOperation: false
+  disableGenerateOperation: true
```

```suggestion
  disableGenerateOperation: false
```
We do have `--operation=generate` in audit pod by default.